wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -1,5 +1,5 @@
-{% macro TimeSeriesComponent(site_no) -%}
-    <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_no }}">
+{% macro TimeSeriesComponent(site_no, latitude, longitude) -%}
+    <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ site_no }}" data-latitude="{{ latitude }}" data-longitude="{{ longitude }}">
         <div class="loading-indicator-container"></div>
```

Comment:
This is going to require some changes to the image server to pass this in, also.. but could be done as a separate ticket.

Reply:
I'll write a separate ticket for this.